quickstart/101-azurerm_dns_zone/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=0.12"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>2.0"
```

Comment:
Reason for 2.0? if not, should update to more current version requirement.

Reply:
I believe TomF generated the TF code using older versions of TF samples. I've been manually changing these since the third sample, but this sample was the first one so it hasn't been changed yet.

Reply:
Fixed.